doc/getting_started/setup_fpga.md

```diff
@@ -103,62 +107,72 @@ This behavior is helpful to create reproducible builds and avoids Vivado modifyi
 
-```console
-# Only create Vivado project directory by using FuseSoC directly (skipping Bazel invocation).
+**Only create Vivado project directory by using FuseSoC directly (skipping Bazel invocation).**
+```sh
 cd $REPO_TOP
-fusesoc --cores-root . run --flag=fileset_top --target=synth --no-export --setup lowrisc:systems:chip_earlgrey_cw310
+fusesoc --cores-root . run --flag=fileset_top --target=synth --no-export --setup lowrisc:systems:chip_earlgrey_${BOARD}
 ```
 
 You can then navigate to the created project directory, and open Vivado
-```console
+```sh
 . /tools/Xilinx/Vivado/{{#tool-version vivado }}/settings64.sh
-cd $REPO_TOP/build/lowrisc_systems_chip_earlgrey_cw310_0.1/synth-vivado/
-vivado
+cd $REPO_TOP/build/lowrisc_systems_chip_earlgrey_${BOARD}_0.1/synth-vivado/vivado
 ```
 
 Finally, using the Tcl console, you can kick off the project setup with
-```console
-source lowrisc_systems_chip_earlgrey_cw310_0.1.tcl
+```sh
+source lowrisc_systems_chip_earlgrey_${BOARD}_0.1.tcl
 ```
 
-## Connecting the ChipWhisperer CW310 board
+## Connecting the ChipWhisperer FPGA board:
+
+### CW310 board
 
-The ChipWhisperer CW310 board supports different power options.
+The CW310 board supports different power options.
 It is recommended to power the board via the included DC power adapter.
 To this end:
 1. Set the *SW2* switch (to the right of the barrel connector) up to the *5V Regulator* option.
-1. Set the switch below the barrel connector to the right towards the *Barrel* option.
-1. Set the *Control Power* switch (bottom left corner, *SW7*) to the right.
-1. Ensure the *Tgt Power* switch (above the fan, *S1*) is set to the right towards the *Auto* option.
-1. Plug the DC power adapter into the barrel jack (*J11*) in the top left corner of the board.
-1. Use a USB-C cable to connect your PC with the *USB-C Data* connector (*J8*) in the lower left corner on the board.
+2. Set the switch below the barrel connector to the right towards the *Barrel* option.
+3. Set the *Control Power* switch (bottom left corner, *SW7*) to the right.
+4. Ensure the *Tgt Power* switch (above the fan, *S1*) is set to the right towards the *Auto* option.
+5. Plug the DC power adapter into the barrel jack (*J11*) in the top left corner of the board.
+6. Use a USB-C cable to connect your PC with the *USB-C Data* connector (*J8*) in the lower left corner on the board.
+### CW340 board
+
+The CW340 board should be powered via the included DC power adapter.
+To this end:
+1. Set the *Control Power* switch (top left corner, *SW7*) to the right.
+2. Ensure the *Tgt Power* switch (center of the board) is set to the right towards the *Auto* option.
+3. Plug the DC power adapter into the barrel jack (*J11*) in the top left corner of the board.
+4. Use a USB-C cable to connect your PC (*host*) with the *USB-C* connector (*J28*) in the lower left corner on the board.
```

Comment:
This guide show say something about the serial switches (JP1-4). This impacts the name of the serial devices below.

Reply:
done